src/pykiso/interfaces/simple_auxiliary.py

```diff
@@ -45,7 +45,11 @@ def __new__(cls, *args, **kwargs):
             add_logging_level("INTERNAL_DEBUG", logging.DEBUG + 1)
         return super(SimpleAuxiliaryInterface, cls).__new__(cls)
 
-    def __init__(self, name: str = None, activate_log: List[str] = None) -> None:
+    def __init__(
+        self,
```

Comment:
Did you reformat this or is it caused by the black version upgrade?

Reply:
Added a useless new param and then deleted it, the reformat is a legacy that I forgot to remove. Done now